pywbem_mock/_wbemconnection_mock.py

```diff
@@ -449,7 +450,117 @@ def add_cimobjects(self, objects, namespace=None):
                 except KeyError:
                     self.qualifiers[namespace] = NocaseDict({qual.name: qual})
             else:
-                assert False, 'add_cimobjects. %s not valid.' % type(obj)
+                assert False, 'Object to add_cimobjects. %s invalid type' \
+                              % type(obj)
+
+    def subscribe_method(self, classname, methodname, method_callback,
```

Comment:
I don't like the name `subscribe_method()`, it is too close to indication subscription. In the mock support, we have used "add" for CIM objects, and what we add here is a "method callback" (at least if we go with the corresponding parameter name). So how about renaming this method to `add_method_callback()`?

Reply:
Agreed.